gtsfm/densify/mvs_patchmatchnet.py

```diff
@@ -0,0 +1,280 @@
+"""Inference methods for GtsfmData in PatchmatchNet
+
+Authors: Ren Liu
+"""
+import time
+from typing import Dict, List
+
+import numpy as np
+import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader
+
+from gtsfm.common.gtsfm_data import GtsfmData
+from gtsfm.common.image import Image
+from gtsfm.densify.mvs_base import MVSBase
+import gtsfm.densify.mvs_utils as mvs_utils
+from gtsfm.densify.patchmatchnet_data import PatchmatchNetData
+from gtsfm.utils import logger as logger_utils
+from thirdparty.patchmatchnet.models.net import PatchmatchNet
+import thirdparty.patchmatchnet.utils as patchmatchnet_utils
+import thirdparty.patchmatchnet.eval as patchmatchnet_eval_funcs
+
+logger = logger_utils.get_logger()
+
+torch.backends.cudnn.benchmark = True
+PATCHMATCHNET_WEIGHTS_PATH = "thirdparty/patchmatchnet/checkpoints/model_000007.ckpt"
+
+# all default values are assigned by Wang et al. https://github.com/FangjinhuaWang/PatchmatchNet/blob/main/eval.py
+DEFAULT_BATCH_SIZE = 1
+DEFAULT_VIEW_NUMBER = 5
+DEFAULT_WORKERS_NUMBER = 4
+
```

Comment:
inline 1-liner explaining what `DEFAULT_GEOMETRIC_PIXEL_THRESH` means mathematically would be great